week1/task3/index.html

```diff
@@ -0,0 +1,76 @@
+<!-- week1-3 웨비의 사진관 -->
+
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>week1 과제3 웨비의사진관</title>
+  <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <header>✨여니꾸✨이모저모✨</header>
```

Comment:
유진언니 리뷰에 남긴것처럼 ! 
header 태그에 제목 요소가 포함되면 더 시맨틱한 구조가 될 수 있다구 합니당
따라서 
```suggestion
  <header><h1>✨여니꾸✨이모저모✨</h1></header>
```
이렇게 구조를 만들수도 있어용 !

헤딩 요소의 위계도 잘 생각해서 구조를 잡아주면 웹 접근성 측면에서도 더 이점을 가질수 있을것 같네용